### PR TITLE
Translations

### PR DESCRIPTION
**Pull request by anhiga**
* Added support for article translations, indepently from blog general language.
* It has to be tested on mobile platforms.
* Main issue is that translations don't keep same folder structure as articles.

### CODE REVIEW

File: theme/templates/article.html [hunk 1]
```diff
@@ -3,6 +3,20 @@
 {% block headerimg %}{% if article.headerimg %}{{ article.headerimg }}{% else %}{{ DEFAULT_HEADER_BG }}{% endif %}{% endblock %}
 
 {% block content %}
+
+{% if article.translations %}
+    <ul class="active" style="padding-left: 10px;float: left;color: #777">
```
Comment (Juanlu001) [inline theme/templates/article.html]: Why `<ul>`?

File: theme/templates/article.html [hunk 2]
```diff
@@ -3,6 +3,20 @@
...
+        Also available in:
+    </ul>
+    {% for lang_url in article.translations %}
+        <ul class="active" style="padding-left: 20px;float:left;">
+        <a href={% if not lang_url.in_default_lang %}"{{ SITEURL }}/{{ lang_url.slug }}-{{ lang_url.lang }}"
+            {% elif article %}"{{ SITEURL }}/{{ lang_url.url }}"
+            {% endif %} title="{{ lang_url.lang }}">{{ lang_url.lang }}</a>
+        </ul>
+    {% endfor %}
+{% endif %}
```
Comment (Juanlu001) [inline theme/templates/article.html]: Some CSS issue, the language selector is not cleared.

**Comment by Juanlu001** [comment]
screenshot_20170721-175616

**Comment by Juanlu001** [comment]
I don't see changes, do you?

**Comment by anhiga** [comment]
Changes are only noticeable on mobile, I think.

photo_2017-07-22_15-35-47

**Comment by Juanlu001** [comment]
Oh! I tried to refresh the cache but it didn't work. Merging!

**Pull request merged**